docs/user/intelmq-api.rst

```diff
@@ -0,0 +1,112 @@
+.. 
+   SPDX-FileCopyrightText: 2020 Birger Schacht
+   SPDX-License-Identifier: AGPL-3.0-or-later
+
+###########
+Intelmq Api
+###########
+
+intelmq-api is a `hug <http://hug.rest>`_ based API for the `intelmq <https://github.com/certtools/intelmq/>`_ project.
+
+**********************************
+Installing and running intelmq-api
+**********************************
+
+``intelmq-api`` requires at least `Python 3.6 <https://www.python.org/downloads/release/python-360/>`_.
+
+Install ``intelmq-api`` using your preferred package installation mechanism (``pip``, ``apt``, ``yum``, ``dnf``...).
+
+* ``pip install intelmq-api``
+* ``apt install python3-intelmq-api``
+* ``yum install intelmq-api``
+* ``zypper install intelmq-api``
+
+The ``intelmq-api`` packages ship a configuration file in ``/etc/intelmq-api/config.json`` and a virtualhost configuration file for Apache 2 in ``/etc/intelmq-api/apache-virtualhost.conf``.
```

Comment:
That file needs to be adapted with the correct path. E.g. for an installation via pip on buster, the path to intelmq-api is `/usr/local/lib/python3.7/dist-packages/intelmq_api/`.

Reply:
Ah, damn- I've updated the paragraph and tried to explain it, I hope that covers most cases